packages/stryker/stryker.conf.js

```diff
@@ -6,6 +6,9 @@ module.exports = function (config) {
     reporter: ['progress', 'clear-text'],
     coverageAnalysis: 'perTest',
     logLevel: 'info',
-    plugins: ['stryker-mocha-runner']
+    plugins: ['stryker-mocha-runner'],
+    mutationOptions: {
+      proportionOfMutantsToUse: 0.1
```

Comment:
I like this grouping in `mutationOptions`. If we later add the ignoring of certain mutator types we can add that here as well.

Maybe we can call it it:

```
mutantOptions: {
   reduce: 0.9
}
```

What would feel the most natural to you?